.github/workflows/load-tests-job-configs/config_ParDo_Go_Batch_200_Iterations.txt

```diff
@@ -0,0 +1,35 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+--job_name=load-tests-go-dataflow-batch-pardo-2-
+--project=apache-beam-testing
+--region=us-central1
+--temp_location=gs://temp-storage-for-perf-tests/loadtests
+--staging_location=gs://temp-storage-for-perf-tests/loadtests
+--influx_measurement=go_batch_pardo_2
+--influx_namespace=dataflow
+--input_options=''{\"num_records\":20000000,\"key_size\":10,\"value_size\":90}''
+--iterations=200
+--number_of_counter_operations=0
+--number_of_counters=0
+--num_workers=5
+--autoscaling_algorithm=NONE
+--environment_type=DOCKER
+--environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest
+--influx_db_name=beam_test_metrics
+--influx_hostname=http://10.128.0.96:8086
```

Comment:
there are lots of common options - I think we need a load test builder that setup common options, like currently Jenkins counterpart has.

Reply:
Done